packages/spindle-ui/src/Toast/Toast.css

```
@@ -10,58 +12,135 @@
   z-index: var(--Toast-z-index);
 }
 
-.spui-Toast--content {
-  background-color: var(--color-surface-accent-primary-light);
+.spui-Toast-content {
+  align-items: center;
   border-radius: 40px;
```

今回の変更とは関係ありませんが、figma見てたらデザインと違ったので一応コメントします 🙇🏻 
```suggestion
  border-radius: 52px;
```